CHANGELOG.md

```diff
@@ -5,88 +5,94 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 
 ## Swift 5.7
 
+* [SE-0326][]:
+
+  Parameter and result types can now be inferred from the body of multi-statement closures.
```

Comment:
I think the first sentence should start from something like `Type-checking behavior of multi-statement closures has been unified with that of single-expression closures.`